code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -411,12 +412,18 @@
 		H.verbs -= ability
 	return
 
-// Do species-specific reagent handling here
-// Return 1 if it should do normal processing too
-// Return 0 if it shouldn't deplete and do its normal effect
-// Other return values will cause weird badness
+/*
+* This proc is used if you want to adjust HOW a species handles their reagents, not what the reagents do directly.
+* If you want to adjust what a reagent does to a certain species, override the appropriate on_species_life proc on the reagent. (on_tajaran_life, on_vox_life, etc)
+* By default, this simply calls the species' respective on_species_life proc on the provided reagent.
+* However, if you wanted to do things like adjust the rate at which a species metabolizes their reagents, you can easily adjust this proc to support that on that species.
+* You even can make it so that a species will only metabolize the reagents if certain criteria are met, such as being over 50% nutrition or below normal body temp, if you wanted.
+* Just make sure you still use the call proc used below in order to call the proper procs, or else you might find your changes doing nothing.
+*/
 /datum/species/proc/handle_reagents(var/mob/living/carbon/human/H, var/datum/reagent/R)
```

Comment:
As the comments directly above it state, the proc is left so you can mess with HOW reagents are handled on the species level (not their effects). For example, to simulate a lower metabolism, you could make the reagent's on_species_life call every 3rd cycle